ff_dir.c

```diff
@@ -2875,24 +2875,31 @@ FF_Error_t FF_ExtendDirectory( FF_IOManager_t * pxIOManager,
 } /* FF_ExtendDirectory() */
 /*-----------------------------------------------------------*/
 
-static const uint8_t forbiddenChrs[] =
+/* *INDENT-OFF* */
```

Comment:
Doesn't look like  INDENT-OFF is doing anything here.

Reply:
No. It is needed.